cmd/runstatus_incident_remove.go

```diff
@@ -8,9 +8,8 @@ import (
 	"github.com/spf13/cobra"
 )
 
-// removeCmd represents the remove command
 var runstatusIncidentRemoveCmd = &cobra.Command{
-	Use:     "remove [page name] <incident name | id>",
+	Use:     "remove [PAGE-NAME] INCIDENT-NAME|ID",
```

Comment:
To stay consistent across other commands:
```suggestion
	Use:     "remove [PAGE] INCIDENT-NAME|ID",
```